src/ScottPlot5/ScottPlot5 Sandbox/Sandbox.WPF/MainWindow.xaml.cs

```diff
@@ -8,7 +9,14 @@ public partial class MainWindow : Window
     public MainWindow()
     {
         InitializeComponent();
-        WpfPlot1.Plot.Add.Signal(Generate.Sin());
-        WpfPlot1.Plot.Add.Signal(Generate.Cos());
+        //WpfPlot1.Plot.Add.Signal(Generate.Sin());
```

Comment:
I assume this was just for testing, but I think this file should be returned to its original state

Reply:
I'll revert this and move this demo code into a cookbook recipe 👍